### ISSUE TITLE
RStudio's pandoc not detected correctly on some M1 Macs

### ISSUE THREAD

**Issue by joelostblom** [UBC-MDS/UBC-MDS.github.io-jekyll#122]
https://ubc-mds.slack.com/archives/C24GEJS11/p1631044163460800 One other student reported missing pandoc too, also on an M1 mac I believe. It is strange because R Studio bundles pandoc and there should be no need for us to test this separately. It might be something related to Rstudio bundled programs are not being on put correctly on the path for some M1 macs... not sure at this point but recorded this in an issue.

**Comment by joelostblom** [comment]
Consider adding a check for pandoc to easier detect this issue.

https://bookdown.org/yihui/rmarkdown-cookbook/install-pandoc.html

**Comment by ttimbers** [comment]
I am dealing with this as well for M1 Macs in DSCI 310 as well. We are asking them to separately install install pandoc for now.

**Comment by flor14** [comment]
The same here trying the installation in a Mac M2. I added Pandoc installation to the instructions for Mac in #139

**Comment by flor14** [comment]
I will close the issue as installing pandoc fix this, but please feel free to open it if you think it is necessary

**Comment by joelostblom** [comment]
I looked into this more and I think the reason that pandoc is found on other OSes than M1 is actually because it is added to path by conda, but for some reason it is not on M1 macs. This is not necessarily better, since it means that RStudio is using a different version of pandoc than when exporting from R from the terminal, which can lead to all sorts of issues that are hard to detect. I opened https://github.com/rstudio/rstudio/issues/11760 for RStudio to fix this so that R always sees the RStudio binaries even when run from terminal.

In the meantime, I think we should explicitly mention this if we teach student how to export Rmarkdown documents from the terminal/R Scripts (not sure if this is part of any course @flor14 @ttimbers ?), and show them how to set which version of pandoc to use for export by setting `rmarkdown::find_pandoc(dir = '/usr/bin/rstudio/bin/quarto/bin/tools', cache=F)` in their scripts. I think this is also a better solution than installing a separate version on M1 macs, for the same reason as above. Adding the RStudio pandoc binary to path in `~/.bashrc` is another option, but that would shadow any explicit installation of pandoc, so it is also not desirable. Since the RStudio pandoc binary is an RMarkdown specific modification, I think the fix should be as well.

I opened https://github.com/UBC-MDS/UBC-MDS.github.io/pull/143 to fix how we are checking this during the setup.
